packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -546,6 +546,27 @@ describe('src/cy/commands/actions/click', () => {
       cy.getAll('span2', 'focus click mousedown').each(shouldNotBeCalled)
     })
 
+    // https://github.com/cypress-io/cypress/issues/28788
+    it('no click when element is disabled', () => {
+      const btn = cy.$$('button:first')
+      const span = $('<span>foooo</span>')
+
+      attachFocusListeners({ btn, span })
+      attachMouseClickListeners({ btn, span })
+      attachMouseHoverListeners({ btn, span })
+
+      btn.html('')
+      btn.attr('disabled', true)
+      btn.append(span)
+
+      cy.get('span').click()
```

Comment:
Currently this is clicking all the spans on the page, which is about 38 of them. I think we just want to click the span inside the button?
```suggestion
      cy.get('button:first span').click()
```